src/main/java/com/conveyal/r5/otp2/rangeraptor/PerformTransitStrategy.java

```diff
@@ -0,0 +1,34 @@
+package com.conveyal.r5.otp2.rangeraptor;
+
+import com.conveyal.r5.otp2.api.transit.TripPatternInfo;
+import com.conveyal.r5.otp2.api.transit.TripScheduleInfo;
+import com.conveyal.r5.otp2.rangeraptor.transit.TripScheduleSearch;
+
+
+/**
+ * Perform the transit part of the routing on behave of the {@link RangeRaptorWorker}.
```

Comment:
Could we add a bit more detail on this class? I'm not sure what a transit strategy is (and it might be worth differentiating from passenger boarding strategies [e.g. https://espace.library.uq.edu.au/view/UQ:20ab185])

Reply:
I just updated the comment, and renamed the interface for clarity. We're planning a renaming session where we'll go through and change a lot of identifiers and class names for clarity. Hopefully in the same PR with Javadoc updates.